frontend/src/components/Candidate-Decider/ResponsesPanel.tsx

```diff
@@ -1,13 +1,60 @@
+import { useEffect, useState } from 'react';
+import { Button, Form, Input, Radio } from 'semantic-ui-react';
+
 type Props = {
   headers: string[];
   responses: string[];
+  rating: number;
+  handleRatingChange: (id: number, rating: number) => void;
+  comment: string;
+  handleCommentChange: (id: number, comment: string) => void;
+  currentCandidate: number;
 };
 
-const ResponsesPanel: React.FC<Props> = ({ headers, responses }) => {
+const ratings = [
+  { value: 1, text: 'No', color: 'red' },
+  { value: 2, text: 'Unlikely', color: 'orange' },
+  { value: 3, text: 'Maybe', color: 'yellow' },
+  { value: 4, text: 'Strong Maybe', color: 'green' },
+  { value: 5, text: 'Yes', color: 'green ' },
+  { value: 0, text: 'Undecided', color: 'grey' }
```

Comment:
will try to style the radio buttons to be the associated color but couldn't figure that out for now